apps/company-landing/app/blog/(posts)/moose-launch/@breadcrumbs/page.tsx

```diff
@@ -1,4 +1,4 @@
-import { Text } from "@514labs/design-system/typography";
+import { Text } from "@514labs/design-system-components/typography";
```

Comment:
Interesting there's now a design-system-components which is separate from the other design-system.

Reply:
Yep, splitting the two libraries and making base re-usable across mono repos and components reusable in a mono-repo